CONTRIBUTING.md

```diff
@@ -99,6 +99,9 @@ The [benchstat](https://godoc.org/golang.org/x/perf/cmd/benchstat) tool is conve
 The special notation "Fixes #12345" associates the change with issue 12345 in the tRPC-Go issue tracker.
 When this change is eventually applied, the issue tracker will automatically mark the issue as fixed.
 
+- If there is a corresponding issue, add either `Fixes #12345` or `Updates #12345` (the latter if this is not a complete fix) to this comment
```

Comment:
https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

<img width="929" alt="image" src="https://github.com/trpc-group/trpc-go/assets/53028871/7c37385f-9e81-4905-8b09-602e32b44d0e">